composer.json

```diff
@@ -55,7 +55,8 @@
         "psr/cache-implementation": "To use metadata caching.",
         "symfony/cache": "To have metadata caching when using Symfony integration.",
         "symfony/config": "To load XML configuration files.",
-        "symfony/twig-bundle": "To use the Swagger UI integration."
+        "symfony/twig-bundle": "To use the Swagger UI integration.",
+        "swagger-api/swagger-ui": "^2.2"
```

Comment:
Interesting, but yeah I followed @teohhanhui suggestion to move out swagger-api from dependencies. Still, I'm not sure if `api-platform/core` should depend on it or not. We may fix those classes IMO if swagger is not installed. If we want it to be integrated, we just have to move this back to the dependencies. WDYT @dunglas ?